src/vm/dllimportcallback.h

```diff
@@ -210,9 +210,11 @@ class UMThunkMarshInfo
                                             // On x86, NULL for no-marshal signatures
                                             // On non-x86, the managed entrypoint for no-delegate no-marshal signatures
     UINT32            m_cbActualArgSize;    // caches m_pSig.SizeOfFrameArgumentArray()
+    UINT16            m_cbRetPop;           // stack bytes popped by callee (for UpdateRegDisplay)
```

Comment:
I've just noticed that these three fields should be present for x86 only (and the offsets for FEATURE_PAL only on top of that), but you have added it for all architectures.